src/wallet/rpcwallet.cpp

```diff
@@ -1093,6 +1141,9 @@ UniValue sendmany(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_CLIENT_P2P_DISABLED, "Error: Peer-to-peer functionality missing or disabled");
     }
 
+    if (!IsDeprecatedRPCEnabled("accounts") && LabelFromValue(request.params[0]) != "") {
```

Comment:
Remove `LabelFromValue` because it raises `JSONRPCError(RPC_WALLET_INVALID_LABEL_NAME, "Invalid label name")`. I suggest to just `... && !request.params[0].get_str().empty()`.

Reply:
agree. Thanks